lldb/test/API/functionalities/reverse-execution/TestReverseContinueBreakpoints.py

```diff
@@ -0,0 +1,151 @@
+import lldb
+import time
+import unittest
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+from lldbsuite.test.gdbclientutils import *
+from lldbsuite.test.lldbreverse import ReverseTestBase
+from lldbsuite.test import lldbutil
+
+
+class TestReverseContinueBreakpoints(ReverseTestBase):
+    NO_DEBUG_INFO_TESTCASE = True
```

Comment:
You can set this on the base class. I don't think we'd ever want to run multiple debug info variants of these tests.

Reply:
Done.